src/pages/typescript/setting-up-slos-in-express.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: Setting up SLOs with Express
+date: 2023-07-04
+---
+
+import { Callout, Cards, Card } from "nextra-theme-docs"
+
+# Setting up SLOs with an Express
+
+In this guide we'll go over what are Service Level Objectives (SLO), why you would want to adopt them for your alerting, and how you can accomplish that in your Express application.
```

Comment:
This sentence says we'll go over what are SLOS but the next section jumps immediately to _why_ and doesn't describe _what_ they are